src/components/Headline/docs/HeadlinePropsTable.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { PropsTable } from '../../../docs/PropsTable';
+
+export const HeadlinePropsTable = () => {
+    const props = [
+        {
+            name: 'as',
+            type: '"h1" | "h2" | "h3" | "h4" | "h5" | "h6"',
+            defaultValue: 'h1'
+        },
+        {
+            name: 'size',
+            type: '"xxl" | "xl" | "l" | "m" | "s" | "xs"',
+            defaultValue: `depends on the heading level`
```

Comment:
We don't set a default here, this might be confusing for the user. How about we add this note in the `description` field, that the size prop will depend on the headling "as" level.